src/components/explorer/YourGuilds.tsx

```diff
@@ -40,13 +40,8 @@ const YourGuilds = ({ guildsInitial }: Props) => {
   const { account } = useWeb3React()
   const { openWalletSelectorModal } = useWeb3ConnectionManager()
 
-  /**
-   * Fetching all guilds without search params to filter user's guilds out of it.
-   * `?order=members` is the default for the other filtered request too, so only one
-   * request will be sent. There'll be a separate endpoint for it in the future
-   */
-  const { data: allGuilds, isLoading: isGuildsLoading } = useSWR(
-    `/guild?order=members`,
+  const { data: allGuilds, isLoading: isGuildsLoading } = useSWRWithOptionalAuth(
+    `/guild?`, // ? is included, so the request hits v2Replacer in fetcher
```

Comment:
Not that important, but could you please change `IntercomProvider` too, so it uses this cache key?

Reply:
Thanks! Fixed in https://github.com/agoraxyz/guild.xyz/pull/803/commits/1af7a4ad54cc1257ae2172694a056482994be918 if that is what you meant 👍